core/db/migrate/20201118200630_add_previous_type_to_spree_payment_methods.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPreviousTypeToSpreePaymentMethods < ActiveRecord::Migration[5.2]
+  def change
+    add_column :spree_payment_methods, :previous_type, :string
```

Comment:
What about `type_before_removal`? I think it expresses better what this field contains.

Reply:
It sounds better actually, fixed.